src/gt4py/backend/gtc_backend/dace/backend.py

```diff
@@ -292,10 +361,12 @@ class GTCDaceBackend(BaseGTBackend, CLIBackendMixin):
         "alignment": 1,
         "device": "cpu",
         "layout_map": make_x86_layout_map,
-        "is_compatible_layout": x86_is_compatible_layout,
-        "is_compatible_type": x86_is_compatible_layout,
+        "is_compatible_layout": lambda x: True,
```

Comment:
Ahhhh. Enough said :-)

Reply:
yes. we don't force vectorization in the dace backend, it's all purely purely strides based.